device/mellanox/x86_64-nvidia_sn5600_simx-r0/syseeprom.hex

```diff
@@ -0,0 +1,256 @@
+54 6c 76 49 6e 66 6f 00 01 01 ee 22 14 4d 53 4e
```

Comment:
@raphaelt-nvidia , please confirm that it contains NVIDIA platform format

Reply:
It does.

Reply:
OK